orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -4065,7 +4065,7 @@ def test_ortmodule_string_inputs_are_ignored():
     with pytest.warns(UserWarning) as warning_record:
         out = ort_model(x, "hello")
 
-    assert len(warning_record) == 2
+    assert len(warning_record) == 3
```

Comment:
can you explain this change?

Reply:
Good catch! because I added a new warning log to tell users the config of recompute here.

 https://github.com/microsoft/onnxruntime/pull/12852/files#diff-117ceac0be922292ecb1559bbd892ae69e6882ab1e0a35823cf24539dd17b2cfR325 

This test catches all warnings and count the numbers, so with the added one warning, it should be 3.